### PR TITLE
Add a basic demo

### PR DESCRIPTION
**Pull request by justinvdm**
Add a basic demo using `DummyUReportApi`

### CODE REVIEW

**Comment by justinvdm** [comment]
Depends on #14 and #19, but otherwise ready for review.

**Comment by justinvdm** [comment]
Ready for review.

File: src/demo.js [hunk 1]
```diff
@@ -1,4 +1,110 @@
...
+            lang: "en"
+        });
+
+    ureport
+        .ureporters
+        .polls
+        .current.returns({
+            id: "poll_agriculture",
+            language: "en",
+            name: "Aggriculture",
```
Comment (hodgestar) [inline src/demo.js]: Too many `g`s?

**Comment by hodgestar** [comment]
Other than the `g`s, looks good.

**Comment by justinvdm** [comment]
Fixed the typo and also fixed a problem with the demo (a side affect of the simplified way we are 'spying' on call args).

**Comment by hodgestar** [comment]
Ah, only the first `when` matches.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**